test/functional/feature_taproot.py

```diff
@@ -1240,8 +1240,8 @@ def sample_spenders():
     # Create a list of scripts which will be built into a taptree
     scripts = [
         # leaf label, followed by CScript
-        ("encodeable_pushdata1", CScript([OP_DROP, OP_PUSHDATA1, b'aa' * 75])),
-        ("nonstd_encodeable_pushdata1", CScript([OP_PUSHDATA1, b'aa'])),
+        ("2byte_pushdata", CScript([OP_DROP, b'\xaa\xaa'])),
```

Comment:
Maybe call this just `2byte_push`, since there is no pushdata going on here?

Reply:
done